subjects/check_user_name/README.md

```diff
@@ -0,0 +1,63 @@
+## check_user_name
+
+### Instructions
+
+Sometimes it is more desirable to catch the failure of some parts of a program instead of just calling panic.
+
+For this exercise you will have to create a tool that manages users' access levels.
+
+You will have to create an `AccessLevel` enum wich could be `Guest`, `Normal`, `Admin`.
+
+You will also have to create a `User` struct which has:
+
+- elements:
+  - `name`: `String`
+  - `acessLevel`: `enum`
+- associated functions:
+  - `new`: which initializes the struct.
+  - `send_name`: which takes only **self** as argument and returns an **Option<&str>** with `None` if the user is a guest or the `name` if the `AccessLevel` has any of the other options.
+  - `check_user_name`: which takes a `User`, calls `send_name` and returns a **tuple** with `true` and the user `name` if `send_name` returns the name or `false` and `"Error: User is guest"` if not
```

Comment:
By reading your proposal I understood you wanted it as an associated function. After reading the link, I still don't see why you say it shouldn't be an associated function since it's related with the struct and you won't use it without it. 
I'll change it anyway.

Reply:
Done.